src/main/java/de/komoot/photon/nominatim/NominatimUpdater.java

```diff
@@ -57,13 +57,10 @@ public void update() {
                             template.update("update placex set indexed_status = 0 where place_id = ?;", placeId);
 
                             Integer indexedStatus = place.getIndexdStatus();
-                            if (indexedStatus == DELETE || (indexedStatus == UPDATE && rank == MAX_RANK)) {
-                                updater.delete(placeId);
-                                if (indexedStatus == DELETE) {
-                                    deletedPlaces++;
-                                    continue;
-                                }
-                                indexedStatus = CREATE; // always create
+                            if (indexedStatus == DELETE) {
+                                updater.delete(PhotonDoc.getBaseId(place.getOsmType(), place.getOsmId(), place.getOsmKey()));
```

Comment:
I don't quite understand that. We don't have a PhotonDoc instance at this point in the code, we could requery the database and generate one as per the normal flow, so that we could use its getUid method but it would seem to be unnecessarily complicated and in the end not do anything different.